packages/@aws-cdk/aws-codedeploy/package.json

```diff
@@ -161,7 +161,13 @@
       "docs-public-apis:@aws-cdk/aws-codedeploy.EcsApplication.applicationName",
       "docs-public-apis:@aws-cdk/aws-codedeploy.IEcsApplication.applicationArn",
       "docs-public-apis:@aws-cdk/aws-codedeploy.IEcsApplication.applicationName",
-      "props-physical-name:@aws-cdk/aws-codedeploy.CustomLambdaDeploymentConfigProps"
+      "props-physical-name:@aws-cdk/aws-codedeploy.CustomLambdaDeploymentConfigProps",
+      "from-signature:@aws-cdk/aws-codedeploy.EcsDeploymentConfig.fromEcsDeploymentConfigName",
+      "from-signature:@aws-cdk/aws-codedeploy.LambdaDeploymentConfig.fromLambdaDeploymentConfigName",
+      "from-signature:@aws-cdk/aws-codedeploy.ServerDeploymentConfig.fromServerDeploymentConfigName",
+      "no-unused-type:@aws-cdk/aws-codedeploy.IEcsDeploymentConfig",
+      "no-unused-type:@aws-cdk/aws-codedeploy.ILambdaDeploymentConfig",
+      "no-unused-type:@aws-cdk/aws-codedeploy.IServerDeploymentConfig"
```

Comment:
I've looked over these and they seem reasonable from our changes, but want to highlight them just in case.